src/theory/quantifiers/trigger.h

```diff
@@ -44,9 +44,12 @@ class InstMatchGenerator;
 * for quantified formula 
 *   forall xy. ( f( x ) != b => ( P( x, g( y ) ) V P( y, z ) ) )
 *
-* Notice that it is only useful to match f( x ) to terms not in the equivalence class of b,
-* and P( y, z ) to terms in the equivalence class of false, as such instances will always be 
-* redundant/entailed in the current context.
+* Notice that it is only useful to match f( x ) to f-applications not in the equivalence class of b,
+* and P( y, z ) to P-applications in the equivalence class of false, as such instances will always be 
+* redundant/entailed by the ground equalities and disequalities the current context.
```

Comment:
OK, I think I understand, but the wording is still confusing.  I suggest:
...and P( y, z ) to P-applications not in the equivalence class of true, as instantiating with such matches would result in a ground formula that is entailed by...